homepairs/homepairsUI/tests/redux/property-list/PropertyListReducer.test.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+    FetchPropertiesAction,
+    FetchPropertyAction,
+    AddPropertyAction,
+    RemovePropertyAction,
+    Property,
+    PropertyListState,
+    UpdatePropertyAction,
+    SetSelectedPropertyAction,
+} from 'homepairs-types';
+import { PropertyListActions } from 'homepairs-redux-actions';
+import { properties, initialState } from '../../../src/state/property-list/reducer';
+
+const { PROPERTY_LIST_ACTION_TYPES } = PropertyListActions;
+
+const PMProperties: Property[] = [
+    {
+        city: 'San Luis Obispo',
+        tenants: 5,
+        bathrooms: 2,
+        bedrooms: 3,
+        state: 'CA',
+        streetAddress: '200 N. Santa Rosa',
+    },
+    {
+        city: 'Glenndale',
+        tenants: 5,
+        bathrooms: 2,
+        bedrooms: 3,
+        state: 'CA',
+        streetAddress: '200 N. Santa Fey',
+    },
+    {
+        city: 'Dublin',
+        tenants: 5,
+        bathrooms: 2,
+        bedrooms: 3,
+        state: 'CA',
+        streetAddress: '200 New Fields Lane',
+    },
+];
+
+const testPropertyListStateWithoutIndex: PropertyListState = {
+    selectedPropertyIndex: undefined,
+    properties: PMProperties,
+};
+
+const testPropertyListStateWithIndex: PropertyListState = {
+    selectedPropertyIndex: 1,
+    properties: PMProperties,
+};
+
+describe('PropertyList Reducer Test', () => {
+
+    describe('Test action with Invalid Property type', () => {
+        it('Undefined State', () => {
+            const testAction: FetchPropertiesAction = {
```

Comment:
this const testAction is literally repeated for both "it"s. just define it at the top once